src/app/components/cds/media-cards/ItemChannels.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames/bind';
+import Tooltip from '../../cds/alerts-and-prompts/Tooltip';
+import FacebookIcon from '../../../icons/facebook.svg';
+import TwitterIcon from '../../../icons/twitter.svg';
+import TelegramIcon from '../../../icons/telegram.svg';
+import ViberIcon from '../../../icons/viber.svg';
+import LineIcon from '../../../icons/line.svg';
+import WhatsAppIcon from '../../../icons/whatsapp.svg';
+import InstagramIcon from '../../../icons/instagram.svg';
+import CheckChannels from '../../../CheckChannels';
+import styles from './Card.module.css';
+
+const ItemChannels = ({
+  className,
+  sortMainFirst,
+  channels,
+}) => {
+  const tiplines = CheckChannels.TIPLINE;
+
+  const humanTiplineNames = {
+    WHATSAPP: 'WhatsApp',
+    MESSENGER: 'Messenger',
+    TWITTER: 'X (Twitter)',
+    TELEGRAM: 'Telegram',
+    VIBER: 'Viber',
+    LINE: 'Line',
+    INSTAGRAM: 'Instagram',
+  };
+
+  function getTiplineNameFromChannelNumber(channel) {
+    return Object.entries(tiplines).find(item => item[1] === channel.toString())[0];
+  }
+
+  function getHumanNameFromChannelNumber(channel) {
+    const tipline = Object.entries(tiplines).find(item => item[1] === channel.toString())[0];
+    return humanTiplineNames[tipline];
+  }
+
+  function ChannelIcon({ channel }) {
+    const tipline = getTiplineNameFromChannelNumber(channel);
+
+    function tiplineIcon(name) {
+      switch (name) {
+      case 'WHATSAPP': return <WhatsAppIcon style={{ color: 'var(--whatsappGreen)' }} />;
+      case 'MESSENGER': return <FacebookIcon style={{ color: 'var(--facebookBlue)' }} />;
+      case 'TWITTER': return <TwitterIcon style={{ color: 'var(--xBlack)' }} />;
+      case 'TELEGRAM': return <TelegramIcon style={{ color: 'var(--telegramBlue)' }} />;
+      case 'VIBER': return <ViberIcon style={{ color: 'var(--viberPurple)' }} />;
+      case 'LINE': return <LineIcon style={{ color: 'var(--lineGreen)' }} />;
+      case 'INSTAGRAM': return <InstagramIcon style={{ color: 'var(--instagramPink)' }} />;
+      default: return null;
+      }
+    }
```

Comment:
Question/suggestion: What if we move this mapping to https://github.com/meedan/check-web/blob/develop/src/app/CheckChannels.js? We do something similar in https://github.com/meedan/check-web/blob/develop/src/app/CheckFeedLicenses.js.

It can be out of scope for this ticket too - maybe we can just create a small refactoring ticket to abstract that to https://github.com/meedan/check-web/blob/develop/src/app/CheckChannels.js and replace it in other places it's done (like the channels filter).

Reply:
Good idea. Filed CV2-4181 for the refactor.